app/views/groups/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= render partial: "shared/navbar" %>
+
+<div id = "group-show" class= "section-wrap">
+<section>
+  <div>
+    <%= link_to groups_path do %>
+    <% end %>
+    <div class = "icon-tile">
+        <%= image_tag @category.icon %>
+        <p>Group: <%= @category.name %></p>
+    <div>
+    <%= link_to destroy_user_session_path, method: :delete do %>
+    <% end %>
+  </div>
+
+
+
+  <div>
+    <p>
+      Total expenses:
+      <%= number_to_currency(@category.expenses.sum(:amount)) %>
+    </p>
+  </div>
+
+    <div class ="expenses-tiles-wrap">
+    <% @expenses.each do |expense| %>
+      <%= render partial: "expense", locals: { expense: expense } %>
+    <% end %>
+  </div>
+
+  <div  class = "add-btn">
+    <%= link_to "Add New Transaction", new_group_expense_path(@category.id) %>
+  </div>
+</section>
+</div>
```

Comment:
- [ ] It will be great to follow the design given for your transaction page.  If you check below:
This is your transaction page:
![your trans](https://user-images.githubusercontent.com/32361963/227156469-e3e4ccc8-2b41-40e8-9f72-ef25491e9822.png)

This is the design of the transaction page provided in the guidelines: 
![supposed ](https://user-images.githubusercontent.com/32361963/227157237-49884aec-bd1d-4dfe-afac-ea9807269a2c.png)


Following the given design guide is a major requirement before the project can be approved. Kindly fix this